mobile/library/objective-c/EnvoyConfiguration.h

```diff
@@ -47,7 +47,17 @@ NS_ASSUME_NONNULL_BEGIN
 @property (nonatomic, strong) NSDictionary<NSString *, EnvoyStringAccessor *> *stringAccessors;
 @property (nonatomic, strong) NSDictionary<NSString *, id<EnvoyKeyValueStore>> *keyValueStores;
 @property (nonatomic, strong) NSArray<NSString *> *statsSinks;
-
+@property (nonatomic, strong) NSString *rtdsLayerName;
```

Comment:
Is this RTDS / ADS stuff always configured together? Ideally we encapsulate these fields into a more structured `RTDSLayer` type so you can build that layer separately from the engine then call `EngineBuilder.addRTDSLayer(...)`. AKA [Parse, don't validate](https://lexi-lambda.github.io/blog/2019/11/05/parse-don-t-validate/).

Reply:
ADS is also used with CDS, and maybe other xds types in the future.  So it seemed simplest to split it out.